app/code/Magebit/Faq1/Api/Data/QuestionSearchResultsInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magebit\Faq1\Api\Data;
+
+use Magento\Framework\Api\SearchResultsInterface;
+
+/**
+ * Interface that defines getters and setters for Item arrays
+ */
+interface QuestionSearchResultsInterface extends SearchResultsInterface
+{
+    /**
+     * @return QuestionInterface[]
+     */
+    public function getItems();
```

Comment:
Add return type, in this case array, also add a short description for what the class does